### ISSUE TITLE
Form control does not have a label, but has a title

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#369]
On https://dmptool.org/plans/new

CODE EXAMPLE:
<input class="form-control" aria-required="true" aria-label="project-title" data-toggle="tooltip" data-placement="bottom" spellcheck="true" size="250" title="" type="text" name="plan[title]" id="plan_title" data-original-title="If applying for funding, state the project title exactly as in the proposal." style="background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAfBJREFUWAntVk1OwkAUZkoDKza4Utm61iP0AqyIDXahN2BjwiHYGU+gizap4QDuegWN7lyCbMSlCQjU7yO0TOlAi6GwgJc0fT/fzPfmzet0crmD7HsFBAvQbrcrw+Gw5fu+AfOYvgylJ4TwCoVCs1ardYTruqfj8fgV5OUMSVVT93VdP9dAzp

1.1.1 Non-text Content (Level A)
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)
3.3.2 Labels or Instructions (Level A)
